### ISSUE TITLE
Turning off validation notification within TM

### ISSUE THREAD

**Issue by Jorieke** [hotosm/tasking-manager#6614]
**Describe the bug**
The notification for a validated task cannot be turned off. It would be great if you could turn that off, not just as an email, but as well as a notification in the TM. 

**To Reproduce**
I switch off in my settings Tasks validation emails - Receive an email when a task you have contributed to is validated: 
![image](https://github.com/user-attachments/assets/c5eea562-282f-4937-97a4-f73508999c8a)
But I still get notifications in the HOT TM about validated tasks: 
![image](https://github.com/user-attachments/assets/fae4f430-c291-46d2-a532-7cf708b26646)

I think it is the same for the Team Announcement emails, and maybe the others as well. I can however not show you an example for that. 

**Expected behavior**
It would be great if you would turn of the notification to have it both for the email notification as the in TM notifications. 

**Additional context**
Thank you for all your hard work team!

**Comment by manjitapandey** [comment]
Hi @Jorieke , When you turned off the validation email notification via settings page, it is supposed to turn of only email notifications. The TM notifications on notification bar is still sent, and it as expected as per current code. 
Do you expect to turn of tm notification as well??

**Comment by Jorieke** [comment]
Hello @manjitapandey 
That is indeed what I was requesting here. I however see that it might be a bit of a strange request, so let me explain a bit further:
As a project manager with a lot of projects, I struggle with finding easily the messages that are written by people. The more automatic messages are as such just noise for me and make it harder to see if I have actual messages that I need to respond to or follow up on.

**Comment by Patrik-Br** [comment]
I agree with Jorieke. And if there would be any issue with a task that I mapped I still will get a notification if validator leave me a feedback and tag me.
